### DISCUSSION TITLE
Is the onnx's IR the same as the TVM's IR?

### DISCUSSION THREAD

**Discussion by calvin886** [onnx/onnx#3029] [Q&A] [answered]
# Ask a Question

### Question
Explain your question here.
I am recently trying to optimize onnx by using TVM(pytorch-> onnx -> TVM), so I am very interested in "Is the onnx's IR the same as the TVM's IR?"
### Further information
- Relevant Area (*e.g. model usage, backend, best practices, converters, shape_inference, version_converter, training, test, operators*): 

- Is this issue related to a specific model?  
**Model name** (*e.g. mnist*):  
**Model opset** (*e.g. 7*):

### Notes
Any additional information, code snippets.

**Comment by jcwchen** [comment]
Hi @calvin886,
Although I don't know TVM well, I believe basically the IRs of TVM and ONNX are different... The main purpose for TVM is optimization and for ONNX is exchange.

**Comment by gramalingam** [comment, marked as answer]
No, they are the not the same.
